app/scripts/views/components/scatterPlot/scatterPlot.js

```diff
@@ -165,9 +185,39 @@
     content.updateDataForDownload = function(fileType) {
       if (['pdf', 'svg'].indexOf(fileType) !== -1) {
         initCanvasDownloadData();
+      } else if (fileType === 'tsv') {
+        initTsvDownloadData();
       }
     };
 
+    function initTsvDownloadData() {
+      var _title = opts_.title || 'Mutation Count vs. CNA';
+      var _data = ['Patient ID', 'Sample ID', 'Mutation Count', 'CNA'];
+
+      if (groups_.length > 1) {
```

Comment:
I struggled a little bit. If the user just load the page and download the file, should we treatment as all selected?